documentation/API proposals/APIproposal_QualityByDesign_CableLabs.md

```diff
@@ -0,0 +1,16 @@
+| ----------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| API family name                           | Quality by Design (QbD)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| API family owner                          | CableLabs                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| API summary                               | API allowing 3rd party applications to:<br>- report appliction KPIs to network operator<br>- receive QbD score<br>- receive root cause and recommended corrective actions<br>- request service improvements                                                                                                                                                                                                                                                                                                                                                                                         |
```

Comment:
The CPE Management (NetworkAccessManagement) use cases is the only place in the API Backlog that references 3rd party applications.  I don't see a need to limit the scope to 3rd party applications.  This could also be use by the service provider or device's applications.

I'd remove this from the scope, but you include it when you discuss the use cases in the slides or using additional MD docs, like Chris did.

Reply:
got it, I changed all `3rd party application` references to just `application` so it is not limiting